proposals/xxxx-package-manager-local-dependencies.md

```diff
@@ -0,0 +1,72 @@
+# Package Manager Local Dependencies
+
+* Proposal: [SE-XXXX](XXXX-package-manager-local-dependencies.md)
+* Authors: [Ankit Aggarwal](https://github.com/aciidb0mb3r)
+* Review Manager: [TBD](https://github.com/)
+* Status: **Discussion**
+
+## Introduction
+
+This proposal adds support for depending on a local package in the manifest
+file.
+
+## Motivation
+
+Edit mode allows developing on set of packages without first committing and
```

Comment:
Rather than focus on edit mode here, I would suggest motivating this by observing that in some cases the relationship between two packages is most accurately described using a local path.  This could be because the packages are still in "bring-up" mode, because they are so closely linked that it doesn't make sense to version one without the other, etc.  The edit mode can be mentioned but I wouldn't present this as a band-aid on the transient nature of edit mode.